pandas/core/arrays/categorical.py

```diff
@@ -2216,6 +2218,15 @@ def _concat_same_type(self, to_concat):
     def _formatting_values(self):
         return self
 
+    def isin(self, values):
+        from pandas.core.series import _sanitize_array
+        values = _sanitize_array(values, None, None)
+        null_mask = isna(values)
+        code_values = self.categories.get_indexer(values)
```

Comment:
@TomAugspurger is there an EA version of this? (meaning API)  (maybe should raise NotImplementedError which could be caught with a default implementation)

Reply:
There isn't yet, but I was thinking about add it. I actually define it on IPArray, since an IP address can be "in" a network.

I'll open an issue (shouldn't delay the changes here).